pkg/flowkit/services/accounts_test.go

```diff
@@ -40,6 +41,17 @@ import (
 	"github.com/onflow/flow-cli/tests"
 )
 
+func setupMock() (*flowkit.State, *Services, *mocks.Gateway) {
```

Comment:
I added this method to setup since i need to mock differently then you do in setup, if there is a way to get what I want from your way of mocking then please let me know.

Reply:
I think you could just use `setup()` and then override the methods you want to define, no? but if that doesn't work for you it is fine.